aws-android-sdk-auth-userpools/src/main/java/com/amazonaws/mobile/auth/userpools/SignUpActivity.java

```diff
@@ -99,11 +98,6 @@ public void signUp(final View view) {
             return;
         }
 
-        if (password.length() < 6) {
-            showError(getString(R.string.password_length_validation_failed));
-            return;
-        }
-
         final CognitoUserAttributes userAttributes = new CognitoUserAttributes();
```

Comment:
Can we verify if appropriate error message will be shown if password length constraint is not met?